mappings/net/minecraft/entity/projectile/BreezeWindCharge.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_uqkupzdc net/minecraft/entity/projectile/BreezeWindCharge
```

Comment:
#554 calls this `BreezeWindChargeEntity`, and it looks like other projectiles have the entity suffix